twoopstracker/twoops/views.py

```diff
@@ -344,11 +344,14 @@ def get_queryset(self):
         return TweetSearch.objects.filter(owner=self.request.user.userprofile)
 
 
-class AccountsLists(generics.ListCreateAPIView):
+class TwitterAccountsLists(generics.ListCreateAPIView):
     serializer_class = TwitterAccountsListsSerializer
     permission_classes = [
         IsAuthenticated,
     ]
+    filter_backends = (filters.OrderingFilter,)
+    ordering_fields = ["name", "created_at"]
```

Comment:
Just noticed that this `created_at` refers to the field from [TimestampedModelMixin](https://github.com/CodeForAfrica/TwoopsTracker/blob/main/twoopstracker/db/models.py#L7). Is this what we want to filter by?

Reply:
Yes @esirK  i.e. when was the list created.